### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by jbpenrath**
# v2.0.0 is here 🎉

## Changed

- Migrate factories to factory_boy 3.2.0
- Spread body-content block if document content height is smaller than
  the window height
- Remove a xhr request by passing course run information to the
  CourseRunEnrollment widget via data-props
- Make the course run "title" field optional
- Replace deprecated xlink:href svg attribute by href
- Update and include the latest version of translations
- Bump `docs` to version 2.0.0

## Added

- Add an API endpoint to synchronize course runs from e.g. an external LMS
- Add a "code" field on the course model to allow reference & synchronization
- Use custom views to handle errors (400, 403, 404, 500)

## Fixed

- Fix Sentry SDK initialization environment and release parameters
- intl-relativetimeformat polyfill did not use the right locale filename
  when locale was composed of a languageCode identical to countryCode
- Make generate demo site work when the default language is set to "fr"
- Add permission checks to course run admin based on their related course page
- Fix image srcset in organization_detail template

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1134,7 +1136,8 @@ us:
 - finish integrating the missing pages and improve the sandbox environment;
 - test and polish the use of richie as a django app / node dependency.
 
-[unreleased]: https://github.com/openfun/richie/compare/v2.0.0-beta.22...master
+[unreleased]: https://github.com/openfun/richie/compare/v2.0.0...master
+[2.0.0-beta.22]: https://github.com/openfun/richie/compare/v2.0.0-beta.22...v2.0.0
```
Comment (jmaupetit) [inline CHANGELOG.md:1140]:
```suggestion
[2.0.0]: https://github.com/openfun/richie/compare/v2.0.0-beta.22...v2.0.0
```

Comment (jbpenrath) [reply CHANGELOG.md]: Oups 🙏

**Comment by sampaccoud** [comment]
Things we need to do in this release:
- bump the docusaurus doc site to v2.0.0
- update translations

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**